### DISCUSSION TITLE
azurecommunity.pl

### DISCUSSION THREAD

**Discussion by kaluzaaa** [AzureCommunityPL/group#179] [Pomoc]
Czas popchnąć azurecommunity.pl do przodu, czyli sprawić, żeby się pojawiło wreszcie ;-) 
Mamy szablon [Winck](http://themeht.com/winck/index.html). Dostosowuję go właśnie do bycia szablonem na Github Pages.

Pytanie do was jak powinna wyglądać główna strona? Propozycję z prostą makietą i który element z [demo szablonu](http://themeht.com/winck/index.html) użyć bardzo mile widziane.

Wstępna dyskusja była, aby odpalić na tej stronie w rzeczy:

- hub o wydarzeniach i wpisy o ogłoszonych CFP
- blog (ktoś coś chce napisać, a nie ma swojego miejsca, notatki np @aryll i @maciejgos robią takie ze spotkań, inne przydatne wpisy)

**Comment by kamil-mrzyglod** [comment]
Blog to rozumiem coś w stylu medium.com? Mi się idea podoba, tak się tylko(głośno) zastanawiam jakie kryteria by były dla opublikowanych treści. Jakiś community board by recenzował artykuł przed publikacją? 😉 

Na samej głównie to chyba fajnie jakby były wrzutki związane z wydarzeniami / aktywnościami community. Może embedded GH discussions.

**Reply by kaluzaaa** [reply]
@kamil-mrzyglod sądzę, że po porostu review 2 dowolnych osób przy PR

Co do discussions to czekamy na preview api :D

**Reply by kamil-mrzyglod** [reply]
No ten preview jest wskazany & oczekiwany ;)

**Comment by maciejgos** [comment]
Moja propozycja to użycie szablonu http://themeht.com/winck/ltr/blog-listing.html 
Podział na kategorie i tagowanie umożliwi filtrowanie wydarzeń, notatek czy ogólnie jakiś news ze świata Azure.

**Reply by kamil-mrzyglod** [reply]
Ale te filtrowanie to chyba trzeba samodzielnie opędzić, co? ;) Pytanie na ile GH Pages się wpasują w bardziej dynamiczny model i czy tego typu funkcjonalność na samym końcu będzie przydatna w dłuższej perspektywie.

**Reply by maciejgos** [reply]
Obawiam się, że pewnie jest tak jak piszesz. Inna opcja to zrobienie "brutalnie" podstron na dane tematy i opcjonalnie jakieś "promowane/najnowsze" informacje na głównej stronie 😉

**Reply by kaluzaaa** [reply]
@kamil-mrzyglod może nie do końca. i tak będzie trzeba jekyll budować za pomocą action, więc można już to bez problemu zrobić (strony dynamiczne do tagów i kategorii z automatu)

Wychodzi mi takie coś:

![image](https://user-images.githubusercontent.com/4700951/102323619-01e44400-3f81-11eb-84f8-4e01659de34c.png)

albo hero ze slajderem zamiast trzech elementów

**Reply by kaluzaaa** [reply]
@aryll @maciejgos @kmadof czy możecie swoje markdowny z repo grupy wrzucić jako post testowo do repo strony.

- Post do https://github.com/AzureCommunityPL/azurecommunitypl.github.io/tree/master/_posts
- Obrazki do /assets/images/posts
- Przykładowa nazwa pliku 2016-05-20-this-post-demonstrates-post-content-styles.md
- Przykład nagłówku dla pliku z postem:

```
---
layout: post
title: "This post demonstrates post content styles"
categories: 
- junk
tags:
- "jakis tag"
author:
- handel z github
---
```

**Reply by kmadof** [reply]
Hejo. Pewnie, godzinka czy dwie i to wrzucę.

**Reply by kamil-mrzyglod** [reply]
Hmm, wygląda to całkiem spoko :)

**Comment by kaluzaaa** [comment]
Kolejne pytanie - jakie kolory?

- Coś na baize kolorów jasnego portalu Azure
- Coś na bazie kolorów z grafik na https://www.facebook.com/azurecommunitypl/ ?

**Reply by kamil-mrzyglod** [reply]
Optuję za opcją nr 2 bo wydaje mi się, że na samym końcu to i tak nie będzie mocno odbiegać od kolorów portalu ;)
